doc-content/shared-kie-docs/src/main/asciidoc/Workbench/Installation/Installation-section.adoc

```diff
@@ -185,3 +69,5 @@ cn[\\ ]*=[\\ ]*role
 ----
 
 will find a match on role 'admin'.
+
+include::business-central-system-properties-ref.adoc[]
```

Comment:
Is there the chance that this include could create duplication?

Reply:
This **include** is for the community documents.
All the community content is located in **Installation-section.adoc** file so there is no duplication.